include/runtime/instance/memory.h

```diff
@@ -294,16 +320,11 @@ class MemoryInstance {
   /// \param Length the store length to data. Need to <= sizeof(T).
   ///
   /// \returns void when success, ErrCode when failed.
-  template <typename T>
+  template <typename T, uint32_t const Length = sizeof(T)>
```

Comment:
`const` in a template parameter definition is meanless.

Reply:
I have removed the `const`